testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/util/AccountHelper.java

```diff
@@ -32,7 +34,7 @@ public static boolean updatePassword(RealmResource realm, String username, Strin
         }
 
         UserRepresentation userRepresentation = userResult.get();
```

Comment:
All these lines above are no longer necessary, they are executed in `getUserId` method.

Reply:
Thanks for noting this, it's removed.